lmdeploy/serve/vl_async_engine.py

```diff
@@ -61,15 +61,20 @@ async def _get_prompt_input(self,
         results = {}
         input_ids = []
         if len(segs) > 1:
-            images = await self.vl_prompt_template.async_collect_pil_images(
-                prompt)
-            features = await self.vl_encoder.async_infer(images)
-
-            from lmdeploy.vl.templates import MiniCPMVTempateWrapper
-            if isinstance(self.vl_prompt_template, MiniCPMVTempateWrapper):
-                decorated, features = self.vl_prompt_template.update_image_token(  # noqa: E501
-                    decorated, features)
-                segs = decorated.split(IMAGE_TOKEN)
+            # yapf: disable
+            images_with_kwargs = await self.vl_prompt_template.async_collect_pil_images(prompt) # noqa E:501
```

Comment:
what does "pil" in "async_collect_pil_images" indicate?

Reply:
pillow, PIL